grove/pyqaoa/maxcut_qaoa.py

```diff
@@ -27,7 +27,7 @@
 
 
 def print_fun(x):
```

Comment:
This is probably not needed anymore now that there's a native python 3 print function.